db/seeds.rb

```diff
@@ -1,7 +1,15 @@
-# This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
-#
-# Examples:
-#
-#   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
-#   Character.create(name: "Luke", movie: movies.first)
+require 'json'
+
+data_file = File.read("./lib/json_file/exercise.json")
+data_hash = JSON.parse(data_file)
```

Comment:
You can replace `File.read` + `JSON.parse` by `JSON.load`.

[Here are the docs](https://ruby-doc.org/stdlib-3.1.0/libdoc/json/rdoc/JSON.html#method-i-load_file)

If you click on the 🔍 you will see the implementation and understand it better.